test/quantization/test_numeric_suite.py

```diff
@@ -189,14 +257,30 @@ def compare_and_validate_results(float_model, q_model, data):
                     AnnotatedConvBnReLUModel(qengine),
                 ]
                 data = self.img_data[0][0]
-                module_swap_list = [nn.Conv2d, nn.intrinsic.modules.fused.ConvReLU2d]
                 for model in model_list:
                     model.eval()
                     if hasattr(model, "fuse_model"):
                         model.fuse_model()
                     q_model = quantize(model, default_eval_fn, self.img_data)
                     compare_and_validate_results(model, q_model, data)
 
+                # Test Linear
```

Comment:
Suggest in general breaking each of these tests into 4 smaller tests: Conv/ConvBN, Linear, DynamicLinear and Functional.